machine-learning/tutorials/SentimentAnalysis/Program.cs

```diff
@@ -3,102 +3,131 @@
-            // during the learning process.  
             // <Snippet5>
-            var pipeline = new LearningPipeline();
+            var model = Train(mlContext, _trainDataPath);
             // </Snippet5>
 
-            // The TextLoader loads a dataset with comments and corresponding postive or negative sentiment. 
-            // When you create a loader, you specify the schema by passing a class to the loader containing
-            // all the column names and their types. This is used to create the model, and train it. 
+            // <Snippet11>
+            Evaluate(mlContext, model);
+            // </Snippet11>
+            
+            // <Snippet16>
+            Predict(mlContext, model);
+            // </Snippet16>
+
+            // <Snippet21>
+            TrainFinalModel(mlContext);
+            // </Snippet21>
+
+            // <Snippet25>
+            PredictWithModelLoadedFromFile(mlContext);
+            // <Snippet25>
+
+            Console.WriteLine();
+            Console.WriteLine("=============== End of process ===============");
+        }
+
+        public static ITransformer Train(MLContext mlContext, string dataPath)
+        {
+            //Note that this case, loading your training data from a file, 
+            //is the easiest way to get started, but ML.NET also allows you 
+            //to load data from databases or in-memory collections.
             // <Snippet6>
-            pipeline.Add(new TextLoader(_dataPath).CreateFrom<SentimentData>());
+            IDataView dataView =_textLoader.Read(new MultiFileSource(dataPath));
             // </Snippet6>
 
-            // TextFeaturizer is a transform that is used to featurize an input column. 
-            // This is used to format and clean the data.
+            // Create a flexible pipeline (composed by a chain of estimators) for creating/training the model.
+            // This is used to format and clean the data.  
+            // Convert the text column to numeric vectors (Features column) 
             // <Snippet7>
-            pipeline.Add(new TextFeaturizer("Features", "SentimentText"));
-            //</Snippet7>
+            var pipeline = new TextFeaturizingEstimator(mlContext,"SentimentText", "Features")
+             //</Snippet7>
 
-            // Adds a FastTreeBinaryClassifier, the decision tree learner for this project, and 
-            // three hyperparameters to be used for tuning decision tree performance.
-            // <Snippet8>
-            pipeline.Add(new FastTreeBinaryClassifier() { NumLeaves = 50, NumTrees = 50, MinDocumentsInLeafs = 20 });
+            // Adds a FastTreeBinaryClassificationTrainer, the decision tree learner for this project  
+            // <Snippet8> 
+                    .Append(mlContext.BinaryClassification.Trainers.FastTree(label: "Label", features: "Features", numLeaves: 50, numTrees: 50, minDatapointsInLeafs: 20));
```

Comment:
>label: "Label", features: "Features" [](start = 77, length = 36)

These are default values. What's the best practice for docs? Leave them out, or spell them out?

Reply:
I've removed them.